### ISSUE TITLE
Eating anything crashes Pixture

### ISSUE THREAD

**Issue by Wuzzy2** [kaadmy/pixture#49]
If you eat anything (e.g. hold apple in hand and leftclick), you get this:
```
2017-05-15 04:25:46: ERROR[Main]: ServerError: Lua: Runtime error from mod 'mobs' in callback item_OnUse(): /home/wuzzy/.minetest/games/pixture/mods/hunger/init.lua:223: attempt to call field 'save_hunger' (a nil value)
2017-05-15 04:25:46: ERROR[Main]: stack traceback:
2017-05-15 04:25:46: ERROR[Main]: 	/home/wuzzy/.minetest/games/pixture/mods/hunger/init.lua:223: in function 'callback'
2017-05-15 04:25:46: ERROR[Main]: 	/usr/share/minetest/builtin/game/item.lua:382: in function </usr/share/minetest/builtin/game/item.lua:380>
```
Tested in fce957452842ca1c9ae02086dbf57e430d22e4fa.

**Comment by kaadmy** [comment]
Yeah I fixed this already locally but haven't pushed yet :D

**Comment by kaadmy** [comment]
Fixed in d4103c3c1554bf48c5b0024d315bde66ffad2cfc
